src/opnsense/www/js/bootstrap-dialog.js

```diff
@@ -1060,7 +1060,7 @@
                 BootstrapDialog.moveFocus();
             });
 
-            // Backdrop, I did't find a way to change bs3 backdrop option after the dialog is popped up, so here's a new wheel.
+            // Backdrop, I didn't find a way to change bs3 backdrop option after the dialog is popped up, so here's a new wheel.
```

Comment:
This is probably a third party module. I mostly tried to ignore things like this, but occasionally I think I missed a thing. Happy to drop. I've mostly added these components to my todo-list, but it's ever-growing...